MANIFEST.in

```diff
@@ -24,3 +24,7 @@ recursive-include invenio_theme *.svg
 recursive-include tests *.py
```

Comment:
Minor: Can I ask you to remove ``# added by check_manifest.py`` and sort the lines of the files :-)